File: TI1.3 REST_JAVA_CONUNI_GRP3/mavenproject2/src/main/java/ec/edu/arguello/mavenproject2/model/Movimiento.java
package ec.edu.arguello.mavenproject2.model;
import java.util.Date;
import jakarta.xml.bind.annotation.XmlRootElement;

@XmlRootElement(name = "movimiento")
public class Movimiento {

    private String cuenta;
    private int nroMov;
    private Date fecha;
    private String tipo;
    private String accion;
    private double importe;

    public Movimiento() {
    }

    // Getters y Setters
    public String getCuenta() {
        return cuenta;
    }

    public void setCuenta(String cuenta) {
        this.cuenta = cuenta;
    }

    public int getNroMov() {
        return nroMov;
    }

    public void setNroMov(int nroMov) {
        this.nroMov = nroMov;
    }

    public Date getFecha() {
        return fecha;
    }

    public void setFecha(Date fecha) {
        this.fecha = fecha;
    }

    public String getTipo() {
        return tipo;
    }

    public void setTipo(String tipo) {
        this.tipo = tipo;
    }

    public String getAccion() {
        return accion;
    }

    public void setAccion(String accion) {
        this.accion = accion;
    }

    public double getImporte() {
        return importe;
    }

    public void setImporte(double importe) {
        this.importe = importe;
    }
}

File: TI1.7 REST_JAVA_CONDBB_EUREKABANK_GR03/02. CLIMOV/app/src/main/java/ec/edu/espe/rest_java_bank/data/models/Movimiento.kt
package ec.edu.espe.rest_java_bank.data.models

import com.google.gson.annotations.SerializedName

data class Movimiento(
    @SerializedName("cuenta")
    val cuenta: String,
    @SerializedName("nromov")
    val nromov: Int,
    @SerializedName("fecha")
    val fecha: String,
    @SerializedName("tipo")
    val tipo: String,
    @SerializedName("accion")
    val accion: String,
    @SerializedName("importe")
    val importe: Double
)
